com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/IDebugAdapterContext.java

```diff
@@ -101,4 +101,8 @@
     StepFilters getStepFilters();
 
     IStackFrameManager getStackFrameManager();
+
+    boolean isDebugging();
```

Comment:
For interface design, it should keep the order of APIs either in alphabetically or grouping by the meaning.